networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/HobbitsP2PNetwork.java

```diff
@@ -222,11 +248,6 @@ public void close() throws IOException {
 
   @Subscribe
   public void onNewUnprocessedBlock(BeaconBlock block) {
```

Comment:
Add this log message back:
`LOG.log(Level.INFO, "Gossiping new block with state root: " + block.getState_root());`
this actually works out better now.  Printing once per peer was annoying.

Reply:
done